site/en/posts/android-package-signer.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing Android Package Signer Library
+metadesc: Bringing Android package signing to the browser.
```

Comment:
Metadesc should be less of a tag line and more of a short description of the content that follows